<html>

<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<meta name="DESCRIPTION" content="Software authored by Gordon Smyth">
<link rel="stylesheet" type="text/css" href="res.css">
<title>Gordon Smyth: Software</title>
</head>

<body bgcolor="#FFFFFF" text="#000000" link="#000FFF">

<p class="parent">
<a href="index.html">Gordon Smyth Home</a></p>

<h1>Software</h1>

<hr>

<h2>Bioconductor R packages</h2>

<p><a href="http://bioinf.wehi.edu.au/limma">limma</a> (2002&ndash;present) has extensive capabilities for pre-processing and differential expression analysis of microarray and RNA-seq experiments.</p>

<p><a href="https://doi.org/doi:10.18129/B9.bioc.limmaGUI">limmaGUI</a> (2003&ndash;present) provides a menu-driven interface to the limma package for analysing two-colour microarray data [41].
Developed by James Wettenhall and maintained by me.</p>

<p><a href="https://doi.org/doi:10.18129/B9.bioc.affylmGUI">affylmGUI</a> (2004&ndash;present) provides a menu-driven interface to the affy, gcrma, affyPLM and limma packages for analysing Affymetrix microarray data [49].
Developed by James Wettenhall and maintained by me.</p>

<p><a href="https://doi.org/doi:10.18129/B9.bioc.edgeR">edgeR</a> (2008&ndash;present) performs differential analysis of RNA-Seq, ChIP-Seq and deepSAGE experiments. Is able to separate technical from biological variation and to borrow strength between transcripts or tags.</p>

<p><a href="https://doi.org/doi:10.18129/B9.bioc.goseq">goseq</a> (2009&ndash;present) provides functional analysis of gene lists from RNA-Seq experiments, adjusting for gene length bias. Authored by Matt Young and maintained by Nadia Davidson.</p>

<p><a href="https://doi.org/doi:10.18129/B9.bioc.csaw">csaw</a> (2014&ndash;present) performs differential binding analyses of ChIP-seq experiments.
Developed and maintained by Aaron Lun.
See the Bioconductor Newsletter article on <a href="http://www.bioconductor.org/help/newsletters/2015_January/#overview-of-the-csaw-package">An overview of the csaw package</a>.
</p>

<p><a href="https://doi.org/doi:10.18129/B9.bioc.diffHic">diffHic</a> (2015&ndash;present) performs differential interaction analyses of Hi-C experiments.
Developed and maintained by Aaron Lun.
</p>


<h2>CRAN R packages</h2>

<p><a href="http://cran.r-project.org/web/packages/statmod/index.html">statmod</a> (2001&ndash;present). An R package for biostatistical modelling, including REML analyses, Tweedie generalized linear models, comparative growth curve analysis, and limiting dilution analyses.</p>

<p><a href="http://cran.r-project.org/web/packages/dglm/index.html">dglm</a>. Double generalized linear models.</p>

<p><a href="http://cran.r-project.org/web/packages/tweedie/index.html">tweedie</a>. Functions for Tweedie distributions, exponential family distributions with power-variance functions. Maintained by Peter Dunn.</p>

<h2>Web tools</h2>

<p><a href="http://bioinf.wehi.edu.au/software/elda">ELDA</a>. Extreme limiting dilution analysis for comparing stem cell frequencies between populations.</p>

<p><a href="http://bioinf.wehi.edu.au/software/compareCurves">Compare Growth Curves</a>. Permutation test for comparing groups of growth curves.</p>


<h2>S-Plus</h2>

I have authored a wide variety <a href="../s/index.html">statistical modelling functions for S-Plus</a>, although I no longer maintain these.

<h2>Matlab</h2>

<p><a href="http://www.statsci.org/matlab">statbox</a> (1990&ndash;2001) is a statistics toolbox for Matlab, including ordinal logistic regression and linear mixed models.
No longer maintained.</p>


<p>&nbsp;</p>

<hr>

<p>&nbsp;</p>

</body></html>